mtp_noms_ops/templates/security/credits_history_list.html

```diff
@@ -0,0 +1,101 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+{% load security %}
+
+{% block page_title %}{{ view.title }} – {{ block.super }}{% endblock %}
+
+
+{% block inner_content %}
+<header>
+  <h1 class="heading-xlarge">{{ view.title }}</h1>
+</header>
+
+<aside class="mtp-object-list-toggle print-hidden">
+  <a href="{% url 'security:check_list' %}" class="button-secondary">{% trans 'View pending' %}</a>
+</aside>
+
+{% include 'mtp_common/includes/message_box.html' %}
+
+<p class="font-large">
+  {% blocktrans trimmed count count=form.total_count with number=form.total_count|separate_thousands %}
+  {{ number }} credit
+  {% plural %}
+  {{ number }} credits
+  {% endblocktrans %}
+</p>
+
+<div class="mtp-results-list mtp-check-list">
+<table class="mtp-table table-font-xsmall">
+  <thead>
+    <tr>
+      <th>{% trans 'Date sent' %}</th>
+      <th>{% trans 'Debit card (from)' %}</th>
+      <th>{% trans 'Prisoner (to)' %}</th>
+      <th>{% trans 'Amount' %}</th>
+      <th>{% trans 'Decision' %}</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for check in objects %}
+    <tr class="mtp-check-row">
+      <td>
+        {{ check.credit.started_at|date:'SHORT_DATETIME_FORMAT' }}
+      </td>
+      <td>
+        <a href="{{ check.credit|sender_profile_search_url }}" class="mtp-print-url-hidden">{{ check.credit.sender_name }}</a>
+        <br/>
+        {{ check.credit|format_card_number }}
+        &nbsp;
+        {{ check.credit.card_expiry_date }}
+        <br/>
+        {{ check.credit.sender_email }}
+        <br/>
+        {{ check.credit.billing_address.postcode }}
+        <br/>
+        <br/>
+        <strong>{% trans 'Rule:' %}</strong>
```

Comment:
i've said this to justin before, but this can be 1 or more rules… did he say he still wants "Rule" rather than "Rules"?

Reply:
Haven't had a chance to speak to Justin today, but makes sense to have Rule and Rules depending on the number of rules.

Reply:
yeah you could do that tho i can't recall if api sends list of rules (no need to add just for the sake of this one letter tho!). personally, i'd lean towards Rules, but don't care enough to push for anything!

Reply:
The API currently adds the prefix for this anyway I think... at least that's how I found it in the other branch - was personally leaving it as it currently works to address properly later

Reply:
Yeah I'd lean towards Rules also. Ill implement Rules for now. Can always change if needed down the line

Reply:
ok

Reply:
Okay Ill leave it as RULE and we can address this later if needed!